### PR TITLE
Letter 'c' from #428

### PR DESCRIPTION
**Pull request by luzpaz**
Continue to separate #428 in to separate PRs for each letter. Also removed/corrected some entries that were not relevant/precise. This PR has been sorted as well.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2119,7 +2271,10 @@ compresion->compression
 compresor->compressor
 compresser->compressor
 comprimise->compromise
+compromize->compromise
+compsite->composit
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: should be compsite->composite

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -1644,14 +1671,18 @@ carreer->career
...
+carryintg->carrying
 cartdridge->cartridge
 Carthagian->Carthaginian
 carthographer->cartographer
 cartilege->cartilage
 cartilidge->cartilage
 cartrige->cartridge
 cas->case
+case-insenstive->case-insensitive
+casesensitve->casesensitive
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: maybe this should be case-sensitive

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -2130,14 +2285,19 @@ computtaions->computations
...
+comute->commute
 conain->contain
 conained->contained
 conainer->container
 conainers->containers
 conaining->containing
 conains->contains
 conainted->contained
+conatain->contain
+conatainer->conaitner
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: should be conatainer->container

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -2367,29 +2593,43 @@ contibutes->contributes
...
 continuting->continuing
 contiuguous->contiguous
+contnt->content
 contol->control
 contoler->controller
 contoller->controller
 contollers->controllers
+contolls->controls
+contols->controls
+contorls->conrtols
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: contorls->controls

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -1680,44 +1712,56 @@ caterpiller->caterpillar
 caterpillers->caterpillars
 cathlic->catholic
 catholocism->catholicism
+catory->factory
 catterpilar->caterpillar
 catterpilars->caterpillars
 catterpillar->caterpillar
 catterpillars->caterpillars
 cattleship->battleship
+cauched->catched
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: cauched->caught

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -1865,40 +1952,56 @@ classe->class, classes,
...
 clearifying->clarifying
+cleck->check
 cleean->clean
 cleint->client
 cleints->clients
 clera->clear, sclera,
 clese->close
 cleses->closes
+clevely->cleverly
+cliensite->clientside
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: cliensite->client-side

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -2025,25 +2151,34 @@ commmenting->commenting
...
+commpessed->compressed
+commpn->common
 commuication->communication
 commuications->communications
 commuinications->communications
 communcation->communication
 communciation->communication
 communiation->communication
 communites->communities
+comobobox->combobox
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: comobobox->combo box

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -2546,28 +2840,51 @@ counterfit->counterfeit
...
 coypright->copyright
+cpation->caption
+cpoy->copy
 cpoy->coy, copy,
+craete->create
+crahses->crashes
+crasher->crash
+crashers->crashes
 crashs->crashes
+crated->created
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: The word _crated_ is valid, this correction should be removed.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: I'll use `crated->created, crated,` JIC ?

Comment (EdwardBetts) [reply codespell_lib/data/dictionary.txt]: Sounds good.

**Pull request merged**